integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/distrox/DistroXImageValidatorTest.java

```diff
@@ -0,0 +1,39 @@
+package com.sequenceiq.it.cloudbreak.testcase.e2e.distrox;
+
+import javax.inject.Inject;
+
+import org.testng.annotations.Test;
+
+import com.sequenceiq.it.cloudbreak.client.DistroXTestClient;
+import com.sequenceiq.it.cloudbreak.context.Description;
+import com.sequenceiq.it.cloudbreak.context.TestContext;
+import com.sequenceiq.it.cloudbreak.dto.distrox.DistroXTestDto;
+import com.sequenceiq.it.cloudbreak.testcase.e2e.ImageValidatorE2ETest;
+
+public class DistroXImageValidatorTest extends ImageValidatorE2ETest {
+
+    @Inject
+    private DistroXTestClient distroXTestClient;
+
+    @Test(dataProvider = TEST_CONTEXT)
+    @Description(
+            given = "there is a running cloudbreak",
+            when = "a valid DistroX create request is sent",
+            then = "DistroX cluster is created")
+    public void testCreateDistroX(TestContext testContext) {
```

Comment:
Since image validation does not use this test currently, I think you can move it back to the DistroxTests class, it does not need to be an imagevalidation test, the only class used for image validation is `InternalSdxDistroxTests `

Reply:
Or you can simply remove this test